clicache/src/impl/ManagedString.hpp

```diff
@@ -40,7 +40,7 @@ namespace Apache
     {
     private:
 
-      IntPtr m_str;
+     IntPtr m_str;
```

Comment:
None of the edits in this file seem to add value.  If that's the case then let's revert this file.

Reply:
Agreed.